pkg/kv/kvserver/replica_proposal_buf.go

```diff
@@ -162,6 +162,7 @@ type proposer interface {
 	// The following require the proposer to hold an exclusive lock.
 	withGroupLocked(func(proposerRaft) error) error
 	registerProposalLocked(*ProposalData)
+	campaignLocked(ctx context.Context)
```

Comment:
a comment might be helpful - most of the callers have a `*RawNode*` in scope, but this method _also_ enqueues to the scheduler.

Reply:
Is that problematic though? This was mostly to avoid having to write the comment all over again at the call site.